src/test/java/com/commercetools/sync/commons/utils/CategoryCustomUpdateActionUtilsTest.java

```diff
@@ -0,0 +1,49 @@
+package com.commercetools.sync.commons.utils;
+
+import com.commercetools.sync.categories.CategorySyncOptionsBuilder;
+import com.commercetools.sync.categories.helpers.CategoryCustomActionBuilder;
+import com.fasterxml.jackson.databind.JsonNode;
+import io.sphere.sdk.categories.Category;
+import io.sphere.sdk.categories.commands.updateactions.SetCustomField;
+import io.sphere.sdk.categories.commands.updateactions.SetCustomType;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.commands.UpdateAction;
+import org.junit.Test;
+
+import java.util.HashMap;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+public class CategoryCustomUpdateActionUtilsTest {
+
+    @Test
+    public void buildTypedSetCustomTypeUpdateAction_WithCategoryResource_ShouldBuildCategoryUpdateAction() {
+        final UpdateAction<Category> updateAction =
+            GenericUpdateActionUtils.buildTypedSetCustomTypeUpdateAction("key", new HashMap<>(), mock(Category.class),
+                new CategoryCustomActionBuilder(), null, Category::getId,
+                categoryResource -> categoryResource.toReference().getTypeId(), categoryResource -> null,
+                CategorySyncOptionsBuilder.of(mock(SphereClient.class)).build()).orElse(null);
+
+        assertThat(updateAction).isNotNull();
+        assertThat(updateAction).isInstanceOf(SetCustomType.class);
+    }
+
+    @Test
+    public void buildRemoveCustomTypeAction_WithCategoryResource_ShouldBuildCategoryUpdateAction() {
+        final UpdateAction<Category> updateAction =
+            new CategoryCustomActionBuilder().buildRemoveCustomTypeAction(null, null);
+
+        assertThat(updateAction).isNotNull();
+        assertThat(updateAction).isInstanceOf(SetCustomType.class);
+    }
+
+    @Test
+    public void buildSetCustomFieldAction_WithCategoryResource_ShouldBuildCategoryUpdateAction() {
+        final UpdateAction<Category> updateAction = new CategoryCustomActionBuilder()
+            .buildSetCustomFieldAction(null, null, "name", mock(JsonNode.class));
+
+        assertThat(updateAction).isNotNull();
+        assertThat(updateAction).isInstanceOf(SetCustomField.class);
```

Comment:
check if the updateaction is a set-action

Reply:
what do you mean?

Reply:
pls, don't duplicate you question/answer on the similar PR comments. The commenter makes you marks where you should take attention. Even if it is duplication of meaning, it has sense to show you all the places in the code where you should take attention.
But this doesn't mean the requestor has to duplicate the same question and flood other developers with useless notifications.

Reply:
@ahalberkamp 8f96c98 1b6054f

Reply:
@andrii-kovalenko-ct sorry ☹️